src/libs/actions/Report.js

```diff
@@ -56,7 +58,19 @@ Onyx.connect({
 
 const typingWatchTimers = {};
 
-// Keeps track of the max sequence number for each report
+/**
+ * Map of the most recent sequenceNumber for a reports_* key in Onyx by reportID.
+ *
+ * There are several sources that can set the most recent reportAction's sequenceNumber for a report:
+ *
+ *     - Fetching the report object
+ *     - Fetching the report history
+ *     - Optimistically creating a report action
+ *     - Handling a report action via Pusher
+ *
+ * Those values are stored in reportMaxSequenceNumbers and treated as the main source of truth about what a report's max
+ * sequenceNumber is.
```

Comment:
How about this phrasing? I stumbled a bit when reading "... the main source of truth about what a report's max sequenceNumber is", and I think the suggestion here is a bit more clear, what do you think?

```suggestion
 * Those values are stored in reportMaxSequenceNumbers and treated as the main source of truth for each report's max sequenceNumber.
```